addons/website_sale/models/sale_order_line.py

```diff
@@ -9,6 +9,7 @@ class SaleOrderLine(models.Model):
 
     name_short = fields.Char(compute='_compute_name_short')
     shop_warning = fields.Char(string="Warning")
+    show_in_cart = fields.Boolean(compute="_compute_show_in_cart")
```

Comment:
I suppose it's for prefetch, but wouldn't it work too with a `_get_website_order_line_domain` or a `_filter_cart_order_line` to avoid a computed field ? :thinking: 

Unless it's really required for a display/storage, I believe avoiding new fields is better (customers won't use it in custom reports/views/..., there won't be cache management and dependencies to handle, ...)

Reply:
Last time I checked (dunno if still the case), even if the field wasn't stored, if a field dependency was impacted, the ORM would still search for all the impacted records to mark the field to be recomputed on those.
This was a good enough reason for me to avoid using those (and the fact that customers can do crap with studio, and afterwards complain to the support because a given field doesn't do what they expect it to do ...).

Is there no way for a simple getter to correctly benefit from the prefetch ?

Reply:
An additional 200 ms on the e-commerce loading time and impacts on all the website pages, I don't think that's a minor detail.

This should have been done before merging these changes. I'm making sure you can make improvements afterward and thus have as many queries as possible within the controller, while also being able to control batch and prefetch. Because if it stays as it is, you'll also have to modify the views, which implies a -u, which we don't want.

I can remove the compute field and tinker with it, since you're worried about people using a field.

I'm doing this before the XP releases to help out; afterward, I won't worry about performance fixes. I'll work on master.